CONTRIBUTING.md

```diff
@@ -18,7 +18,11 @@ The following is a set of guidelines for contributing to the NGINX Ingress Contr
 
 ## Ask a Question
 
-We will have a public forum soon where you can come and ask questions and have a discussion. For now please open an Issue on GitHub with the label `question`.
+To ask the core development team a question please use the [Github Discussions](https://github.com/nginxinc/kubernetes-ingress/discussions) feature.
+
+You can also join our [Community Slack channel](https://community.nginx.org/joinslack) which has a wider NGINX audience.
+
+Please reserve Github issues for specific feature requests and bugs rather than general questions.
```

Comment:
```suggestion
Please reserve Github issues for feature requests and bugs rather than general questions.
```
No need to have the word 'specific', because the nature of feature requests means they're more than likely going to be inherently specific anyway.